myplanet/skeletor
# SKEL-33: Restructure theme

## Testing notes
* Check out branch
* Create a new drupal site from scratch
* Follow instructions in [Starterkit README file](https://github.com/myplanet/skeletor/tree/SKEL-33-theme-restructure/themes/custom/barebones_bootstrap/STARTERKIT). Recommended that you place the theme in Skeletor's `themes/custom` directory
* Enable the theme
* Add a couple of navigation menus
* Enable a Skeletor Component, place it on a new content type, and ensure that it looks like it should.

JIRA story: https://myplanet.jira.com/browse/SKEL-33
